File: C/exer_10.c

/*******1|********2|********3|********4|********5|********6|********7|********8|
'########:'##::::'##:'####:'##:::::::'####::::'###::::'##::: ##::'#######::
 ##.....:: ###::'###:. ##:: ##:::::::. ##::::'## ##::: ###:: ##:'##.... ##:
 ##::::::: ####'####:: ##:: ##:::::::: ##:::'##:. ##:: ####: ##: ##:::: ##:
 ######::: ## ### ##:: ##:: ##:::::::: ##::'##:::. ##: ## ## ##: ##:::: ##:
 ##...:::: ##. #: ##:: ##:: ##:::::::: ##:: #########: ##. ####: ##:::: ##:
 ##::::::: ##:.:: ##:: ##:: ##:::::::: ##:: ##.... ##: ##:. ###: ##:::: ##:
 ########: ##:::: ##:'####: ########:'####: ##:::: ##: ##::. ##:. #######::
........::..:::::..::....::........::....::..:::::..::..::::..:::.......:::
Arquivo: exer_10.C
Autor: Emiliano Costa Junior
Data: 23 de Agosto 2021 
Descrição: Entrar via teclado, com dois valores distintos. Exibir o menor deles.
------------------------------------------------------------------------------*/
#include <stdio.h>  
#include <stdlib.h> 
#include <locale.h>

int main(int argc, char const *argv[])
{
//Inicia
  setlocale(LC_ALL, "Portuguese");
  printf("Exercicio 10:\t");

//Entrada de dados 
  printf("Exibe o menor\n");
  printf("Digite o primeiro valor: ");
  int d_valorA;
  scanf("%d",&d_valorA);
  printf("Digite o segundo valor: ");
  int d_valorB;
  scanf("%d",&d_valorB);

//decisão
	if(d_valorA<=d_valorB)
		printf("Menor valor é %d", d_valorA);
	else
		printf("Menor valor é %d", d_valorB);

//Termina
  return (EXIT_SUCCESS);
}/*FIM MAIN*/
/*Fim de Arquivo-------------------------------------------------------------*/

File: Python/exer10.py
#--------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|
#Emiliano Costa Junior - RA:1260901923008
#FATEC - Automação Industrial
#Programação Aplicada a Automação
#Programa 10 - Seleciona o menor valor digitado
#--------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|
#Descrição: 
#Entrar via teclado, com dois valores distintos. Exibir o menor deles.
#--------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|

#Mensagem de Apresentação
print("Seleciona o menor valor digitado")

#Entrada dados
valor1 = float(input("Digite o valor1: "))
valor2 = float(input("Digite o valor2: "))

#escolhe o maior
if(valor1 < valor2):
    print(f"menor valor é:{valor1: .2f}")
else:
    print(f"menor valor é:{valor2: .2f}")

#Aguarda o usuario finalizar
fim = input("clique qualquer tecla para finalizar!")
#--------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|---------|